core/src/main/kotlin/com/malinskiy/marathon/execution/device/DeviceActor.kt

```diff
@@ -172,6 +173,10 @@ class DeviceActor(private val devicePoolId: DevicePoolId,
         job = async(context, parent = deviceJob) {
             try {
                 device.execute(configuration, devicePoolId, batch, result, progressReporter)
+            } catch (e: DeviceLostException) {
+                logger.error(e) { "Critical error during execution" }
+                returnBatch(batch)
+//                terminate()
```

Comment:
are you sure that it is correct?

Reply:
Yes, for starters I want to just do the same logic, but log that it's a critical error that should lead in the end to device disconnected